build/index.js

```diff
@@ -331,12 +331,12 @@ exports.default = Slideshow;
 /* 4 */
 /***/ (function(module, exports, __webpack_require__) {
 
-exports = module.exports = __webpack_require__(5)(undefined);
+exports = module.exports = __webpack_require__(5)(false);
```

Comment:
thanks for these changes. Just wondering why have you passed in `false` here, as opposed to `undefined`?

Reply:
I only changed the javascript source file and ran "npm run build", so I'm not sure why it is different.